crates/script/src/lib.rs

```diff
@@ -279,6 +279,10 @@ impl ScriptArgs {
                 .as_ref()
                 .is_none_or(|txs| txs.is_empty())
             {
+                if pre_simulation.args.broadcast {
+                    warn!("No transactions to broadcast.");
```

Comment:
please use `sh_warn`

Reply:
I have added, do check it out!

Reply:
looks good, please add the requested test

Reply:
Hello @grandizzy, can I raise a separate PR for the test case for issue #10384 and for this , once both are merged?( as in one pr for testcase for both)

Reply:
please add test per PR, see https://github.com/foundry-rs/foundry/blob/master/CONTRIBUTING.md#adding-tests thank you!

Reply:
sure ,I'll push in a while

Reply:
Added a testcase do check it out @grandizzy